src/panels/lovelace/entity-rows/hui-error-entity-row.ts

```diff
@@ -0,0 +1,45 @@
+import { html, LitElement } from "@polymer/lit-element";
+import { TemplateResult } from "lit-html";
+
+class HuiErrorEntityRow extends LitElement {
+  public entity?: string;
+
+  static get properties() {
+    return {
+      entity: {},
+    };
+  }
+
+  protected render(): TemplateResult {
+    if (!this.entity) {
```

Comment:
I think that we should always render this element, even if no entity set. Make it `${this.enttiy || ''}` below and it's fine

Reply:
I also indirectly meant that we should remove this if, or else it doesn't make sense to default below.